gensim/models/tfidfmodel.py

```diff
@@ -118,6 +118,7 @@ def precompute_idfs(wglobal, dfs, total_docs):
     # not strictly necessary and could be computed on the fly in TfidfModel__getitem__.
     # this method is here just to speed things up a little.
     return {termid: wglobal(df, total_docs) for termid, df in iteritems(dfs)}
+    
```

Comment:
remove this extra line.

Reply:
Various blank lines are the ghosts of `print`s put in during debugging.

Reply:
These are all pep8 errors which will error out your builds. Please use `flake8 gensim/` command as suggested before.